site/src/components/CopyButton/CopyButton.tsx

```diff
@@ -23,36 +24,44 @@ export const Language = {
 /**
  * Copy button used inside the CodeBlock component internally
  */
-export const CopyButton: FC<CopyButtonProps> = ({
-  text,
-  ctaCopy,
-  wrapperStyles,
-  buttonStyles,
-  tooltipTitle = Language.tooltipTitle,
-}) => {
-  const { isCopied, copy: copyToClipboard } = useClipboard(text);
+export const CopyButton = forwardRef<HTMLButtonElement, CopyButtonProps>(
+  (props, ref) => {
+    const {
+      text,
+      ctaCopy,
+      wrapperStyles,
+      buttonStyles,
+      onClick: outsideOnClick,
+      tooltipTitle = Language.tooltipTitle,
+    } = props;
```

Comment:
Generally try to keep prop definitions inlined, but Prettier's auto-formatting made the code super ugly to look at